src/index.jsx

```diff
@@ -0,0 +1,43 @@
+import 'babel-core/polyfill';
+import './css/styles.css'; // add the css for all pages
+import React from 'react';
+import BrowserHistory from 'react-router/lib/BrowserHistory';
+import { Provider } from 'react-redux';
+import { Router, Route } from 'react-router';
+import configStore from './app/Store/configStore';
+
+// may need an App container???
+
+// pages
+var App           = require('./app/Containers/app');
+var Login         = require('./app/Containers/login');
+var FriendManager = require('./app/Containers/friendManager');
+var GiftDetail    = require('./app/Containers/giftDetail');
+var GiftRecs      = require('./app/Containers/recommendedGifts');
+// TODO GiftList View for each friend
+
+const history = new BrowserHistory();
+
+// store
+const store = configStore();
+
+// react routes
+// TODO add route for friendGiftList
+// <Route name="friendGiftList" path="/friends/:friendId/giftlist" handler={giftList} />
+
+React.render(
```

Comment:
set up routes and paths for the entire application
components are the main containers (or views) of the app
pass in the store to the provider and make it available to all the children